### PR TITLE
V1.0

### PR DESCRIPTION
**Pull request by zhengsx**
code and script to be released

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+# data
+dataset/
+
+# reranking
+/examples/reranking/rerank_data
+
+# Cython-generated C++ source files
+/fairseq/data/data_utils_fast.cpp
+/fairseq/data/token_block_utils_fast.cpp
```
Comment (guolinke) [inline .gitignore:129]: fairseq related could be removed.

File: graphormer/model.py [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+        attention_dropout_rate,
+        flag=False,
+        flag_m=3,
+        flag_step_size=1e-3,
+        flag_mag=1e-3,
+    ):
+        super().__init__()
+        self.save_hyperparameters()
+
+        self.head_size = head_size
```
Comment (guolinke) [inline graphormer/model.py:49]: head_size is actually num_head. This name is confusing.

File: .gitignore (hunk 1 above)
Comment (zhengsx) [reply .gitignore]: Done.

File: examples/benchmarking-gnns/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+PNA          | 387.2K  | 0.142      |
+SAN          | 508.6K  | 0.139      |
+Graphormer-Slim   | 489.3K  | **0.122**      |
+
+## Example Usage
+
+```
+[ -z "${exp_name}" ] && exp_name="zinc"
+[ -z "${seed}" ] && seed="1"
+[ -z "${arch}" ] && arch="--ffn_dim 80 --hidden_dim 80 --head_size 8 --dropout_rate 0.1 --n_layers 12 --peak_lr 2e-4 --edge_type multi_hop --multi_hop_max_dist 20"
```
Comment (guolinke) [inline examples/benchmarking-gnns/README.md:28]:
```suggestion
[ -z "${arch}" ] && arch="--ffn_dim 80 --hidden_dim 80 --num-heads 8 --dropout_rate 0.1 --n_layers 12 --peak_lr 2e-4 --edge_type multi_hop --multi_hop_max_dist 20"
```

**Review by guolinke — APPROVED** [review]

**Pull request merged**